manifests/shared/deployments/datadog.yml

```diff
@@ -0,0 +1,14 @@
+---
+releases:
```

Comment:
Quite likely, I don't understand a subtlety of bosh/concourse here.
We specify a git resource for the agent here: https://github.com/alphagov/paas-cf/blob/fc4624c220d341cb8dc86c7435a5c763cd8be612/concourse/pipelines/create-bosh-cloudfoundry.yml#L101-L106
which we then use to create a binary and upload it to bosh.
But then, here, we rely on a manually created release tarball to define the release.
Aren't these achieving the same thing (an agent binary uploaded to bosh) in two different ways?